apps/sms/tools/update_nga_libs.sh

```diff
@@ -12,7 +12,7 @@ bower_path="bower_components"
 
```

Comment:
So I haven't found way to use our shell script to get and "bridgify" minimized and split (service or client only) versions from threads repo (unless we do google closure compiler here like threads lib does). I'll be vooooooting for https://github.com/gaia-components/threads/issues/57 :)

Also I've noticed that minimized versions are ES5 complaint [1], I would say we don't want this in SMS, sine FxOS suports ES6?

[1] https://github.com/gaia-components/threads/blob/master/package.json#L17-L19